src/weights.jl

```diff
@@ -9,6 +9,11 @@ end
 WeightVec{Vec<:RealVector,W<:Real}(vs::Vec,wsum::W) = WeightVec{W,Vec}(vs, wsum)
 WeightVec(vs::RealVector) = WeightVec(vs, sum(vs))
 
+"""
+    weights(vs) -> WeightVec
+
+Construct a weight vector from a given array.
```

Comment:
Could you explain what's a `WeightVec` object? Could essentially use the same docstring for `WeightVec`. I think you can find some material in the manual.